source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -365,6 +415,18 @@ Parameter Details
     - Strings
     - The custom data field partner name you want to use as the destination in a dynamic route node. Typically a phone number in e164 format.
 
+  * - error_prompt_disabled
+    - Boolean
+    - If set to true, no error sound or prompt will play when a caller makes an invalid keypress. If enabled, when a caller makes an invalid keypress, an error sound will play, or you can optionally define a custom error prompt via the parameter custom_error_prompt_text.
+
+  * - keypress_failover_type
+    - String
+    - The failover type to use for a child node of a Menu. "Wrong" for when a wrong keypress is pressed by the caller on any attempt for the parent menu (shown in reporting as keypress "W"). "None" for when there is no keypress by the caller for all attempts for the parent menu (shown in reporting as keypress "N"). Omit this parameter for normal keypresses. See example below.
+
+  * - number_question_type
+    - String
+    - The type of question you want to ask as part of the NumberQuestion node type. This may be "Digits", "Number", "Phone Number" "Date", "Currency", "Time", or Zip Code".
```

Comment:
`"Phone Number"` and `"Zip Code"` should not have spaces -> `"PhoneNumber"`, `"ZipCode"`

(Also zip code is missing a `"` in front)

Reply:
thanks for catching this!